docs/wrapping-react/more-wrapping-examples.md

```diff
@@ -0,0 +1,430 @@
+```
+
+In this example we are wrapping the React Leaflet library from this [NPM package](https://www.npmjs.com/package/react-leaflet).
+
+On line `1` we import the `dynamic` function from Next.js and on line `21` we set `ssr: false`. Lines `4` and `6` use the `dynamic` function to import the `MapContainer` and `TileLayer` components from the `react-leaflet` library. This is done in Reflex by using the `NoSSRComponent` class when defining the component. This is used to dynamically import the `MapContainer` and `TileLayer` components from the `react-leaflet` library. This is done in Reflex by using the `NoSSRComponent` class when defining the component. There is more information of when this is needed on the `Dynamic Imports` section of this [page]({docs.wrapping_react.guide.path}).
+
+It mentions in the documentation that it is necessary to include the Leaflet CSS file, which is added on line `2` in the React code below. This can be done in Reflex by using the `add_imports` method in the `MapContainer` component. We can add a relative path from within the React library or a full URL to the CSS file.
+
+Line `4` defines a functional React component, which on line `8` returns the `MapContainer` which is done in the Reflex code using the `map_container` component.
+
+The `MapContainer` component has props `center`, `zoom`, `scrollWheelZoom`, which we wrap in the `MapContainer` component in the Reflex code. We ignore the `style` prop as it is a reserved name in Reflex. We can use the `rename_props` method to change the name of the prop, as we will see in the React PDF Renderer example, but in this case we just ignore it and add the `width` and `height` props as css in Reflex.
+
+The `TileLayer` component has a prop `url` which we wrap in the `TileLayer` component in the Reflex code.
+
+Lines `24` and `25` defines and exports a React functional component named `Home` which returns the `MapComponent` component. This can be ignored in the Reflex code when wrapping the component as we return the `map_container` component in the `index` function.
+
+
+```javascript
+1 | import dynamic from "next/dynamic";
+2 | import "leaflet/dist/leaflet.css";
+3 | 
+4 | const MapComponent = dynamic(
+5 |   () => {
+6 |     return import("react-leaflet").then((\{ MapContainer, TileLayer }) => {
+7 |       return () => (
+8 |         <MapContainer
+9 |           center=\{[51.505, -0.09]}
+10|           zoom=\{13}
+11|           scrollWheelZoom=\{true}
+12|           style=\{\{ height: "50vh", width: "100%" }}
+13|        >
+14|          <TileLayer
+15|            url="https://\{s}.tile.openstreetmap.org/\{z}/\{x}/\{y}.png"
+16|          />
+17|        </MapContainer>
+18|      );
+19|    });
+20|  },
+21|  \{ ssr: false }
+22| );
+23|
+24| export default function Home() {
+25|   return <MapComponent />;
+26| }
+```
+
+
+```python 
+from reflex.components.component import NoSSRComponent
+from reflex import Var
+
+class MapContainer(NoSSRComponent):
```

Comment:
i think `rx.NoSSRComponent` is also exposed here.

Reply:
changed all of these and all of rx.Component as well